samples/sysbuild/with_mcuboot/boards/stm32h573i_dk.overlay

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2025 STMicroelectronics
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
should probably also be added to the hawkbit sample, so the workaround of https://github.com/zephyrproject-rtos/zephyr/pull/92809 can be reversed

Reply:
this overlay should also be added to the hawkbit sample